client/src/pages/FlightData/tabs/Quick.js

```diff
@@ -96,37 +96,40 @@ const Quick = () => {
+							content={AaltitudeIsGlobal ? AaltitudeGlobal.toFixed(2) + " ft MSL" : Aaltitude.toFixed(2) + " ft AGL"}
+							onClick={() => {setAaltitudeIsGlobal(!AaltitudeIsGlobal)}}
+							style={{ cursor: "pointer" }}
+							title="The plane's altitude. MSL refers to above mean sea level. AGL is the height from the ground where the plane is." />
 						<Box label="Battery (6S)" content={Abattery.toFixed(2) + "V"} />
 					</Row>
 				</Row>
 				<Row style={{ gap: "1rem" }}>
 					<Row>
 						<Box label="Ground Speed"
-							 content={((AspeedIsInKnots ? 0.868976 : 1) * AgroundSpeed).toFixed(2) + (AspeedIsInKnots ? " knots" : " mph")}
-							 onClick={() => {setAspeedIsInKnots(!AspeedIsInKnots)}}
-							 style={{ cursor: "pointer" }} />
+							content={((AspeedIsInKnots ? 0.868976 : 1) * AgroundSpeed).toFixed(2) + (AspeedIsInKnots ? " knots" : " mph")}
+							onClick={() => {setAspeedIsInKnots(!AspeedIsInKnots)}}
+							style={{ cursor: "pointer" }}
+							title="Speed over only the surface of the earth, not taking height into account." />
 						<Box label="Airspeed"
-							 content={((AspeedIsInKnots ? 0.868976 : 1) * Aairspeed).toFixed(2) + (AspeedIsInKnots ? " knots" : " mph")}
-							 onClick={() => {setAspeedIsInKnots(!AspeedIsInKnots)}}
-							 style={{ cursor: "pointer" }} />
+							content={((AspeedIsInKnots ? 0.868976 : 1) * Aairspeed).toFixed(2) + (AspeedIsInKnots ? " knots" : " mph")}
+							onClick={() => {setAspeedIsInKnots(!AspeedIsInKnots)}}
+							style={{ cursor: "pointer" }}
+							title="Speed of the plane overall, including change in altitude." />
 					</Row>
 					<Row>
 						<Box label="Status" content={Astatus} />
-						<Box label="Mode" content={Amode} />
+						<Box label="Mode" content={Amode} title="The flight mode the plane is in, including RTL, Auto, and Manual." />
 					</Row>
 				</Row>
 				<Row style={{ gap: "1rem" }}>
 					<Row>
-						<Box label="Waypoint #" content={"#" + (Awaypoint[0] + 1).toFixed(0)} />
-						<Box label="Distance" content={Awaypoint[1].toFixed(2) + " ft"} />
+						<Box label="Waypoint #" content={"#" + (Awaypoint[0] + 1).toFixed(0)} title="The waypoint number the plane is traveling to." />
+						<Box label="Distance" content={Awaypoint[1].toFixed(2) + " ft"} title="The distance to the next waypoint." />
 					</Row>
 					<Row>
-						<Box label="GPS HDOP" content={Aconnection[0].toFixed(2)} />
-						<Box label="GPS VDOP" content={Aconnection[1].toFixed(2)} />
-						<Box label="Satellites" content={Aconnection[2].toFixed(0)} />
+						<Box label="GPS HDOP" content={Aconnection[0].toFixed(2)} title="Horizontal dilution of precision. The higher, the less accurate the GPS is." />
+						<Box label="GPS VDOP" content={Aconnection[1].toFixed(2)} title="Vertical dilution of precision. The higher, the less accurate the GPS is." />
+						<Box label="Satellites" content={Aconnection[2].toFixed(0)} title="The number of satellites the plane is using." />
```

Comment:
Maybe mention thresholds for proper flight?

4 satellites is the absolute minimum, with 6 being reasonable, 8+ being good, and 10+ being very accurate